### ISSUE TITLE
[benchmark] Xenon on Carbon Sputtering Yields

### ISSUE THREAD

**Issue by drobnyjt** [lcpp-org/RustBCA#235]
**Description**
[A critical review and meta-analysis of xenon-on-carbon sputter yield data](https://doi.org/10.1063/5.0180720), a review paper just published by J. E. Polk, would serve as a great dataset for comparison.

This figure from the paper:
![image](https://github.com/lcpp-org/RustBCA/assets/37962344/1526575f-cd56-4df7-b98a-a2bc82862866)
Shows how well the various included studies agree with one another.

The paper overall also includes discussion about sources of systematic error in sputtering yield measurements that apply generally and to carbon specifically.

**Comment by drobnyjt** [comment]
Note: the sputtering threshold observed in experiment is much lower (<50 eV) than predicted by a few-collision model (100+ eV). [This paper](https://doi.org/10.1016/j.nimb.2009.01.127) demonstrates that the effect is due to the accumulation of xenon in Carbon, which significantly reduces the sputtering threshold to energies seen in experiment. They suggest using an accumulated xenon percentage of 14% to match values seen in experimental low-energy sputtering yields of Xe on C.

**Comment by drobnyjt** [comment]
![Figure_1](https://github.com/lcpp-org/RustBCA/assets/37962344/69f5687e-8612-4c84-b66b-8f1046915eea)
Here's the comparison of RustBCA with experiment for three different Xenon accumulations, 0.0 Xe/C, 0.05 Xe/C, and 0.1 Xe/C. It's obvious that the low sputtering threshold can only be explained by nonzero xenon accumulation, since no other free parameter has as significant an effect.

I ran simulations using Oen-Robinson electronic stopping, per the recommendations in the ACAT paper. I used the following properties for carbon:
```
carbon = {
    'symbol': 'C',
    'name': 'carbon',
    'Z': 6,
    'm': 12.011,
    'n': 1.1331E29,
    'Es': 7.37,
    'Ec': 1.0,
    'Eb': 0.0,
}
```
And the properties for xenon in `scripts/materials.py`. Since I only wanted to measure sputtering yields, I overwrote Ec for both species with the Es of carbon.

I ran all three xenon concentrations using both isotropic and planar surface binding models; isotropic would be more appropriate for an atomically rough surface, whereas planar would be more appropriate for an atomically smooth surface. In this case, I think isotropic surface binding better reproduces the data.

Best agreement is for a xenon accumulation of 10% Xe/C, but agreement is still acceptable for 5% Xe/C. Although there is good agreement between RustBCA and experiment here, there is one caveat and a cautionary tale. The caveat is that, when viewed in log-log form, it can be seen that RustBCA still overestimates the sputtering threshold by about 4-6 eV:
![Figure_2](https://github.com/lcpp-org/RustBCA/assets/37962344/d215d949-cf38-4e69-833a-dfdba7516bef)
This may be due to a lower actual surface binding energy, multi-body effects, or some other phenomenon not captured in the code.

The cautionary tale lies in the importance of accumulated xenon; the sputtering yield curves are very different for a pure carbon target. When dealing with ions with a mass ratio far from one, the accumulation of those ions must be accounted for when calculating sputtering yields.
